I.write-tests/02-remove-vowels/remove-vowels.js

```diff
@@ -1,18 +1,10 @@
 function removeVowels(word) {
-    var characters = word.split('');
-
-    var result = [];
-
-    characters.forEach(function (character) {
-        if (character === 'a' || character === 'o' || character === 'i' || character === 'e'
-            || character === 'u' || character === 'y') {
-            result.push(character)
-        } else {
-            result.push('_')
-        }
-    })
-
-    return result.join('');
+  word = word.toLowerCase();
+  var characters = word.split("");
+  var vowels = ["a", "e", "i", "o", "u"];
+  var result = [];
+  result = characters.filter(letter => vowels.indexOf(letter) < 0).join("");
+  return result;
```

Comment:
Very nice :clap: :tada: 
Can we make use of the `join` function in the `greetPeople` as well? that would make that function much smaller ;)
Also, since you are assigning the variable `result` in one line and returning it in the next, you may as well just do return `characters.filters()` expressions